### ISSUE TITLE
Bug for transformation matrix regularization

### ISSUE THREAD

**Issue by alex-xun-xu** [yanx27/Pointnet_Pointnet2_pytorch#59]
I found the follow bug for transformation matrix regularization. According to the original paper, the transformation matrix is encouraged to be orthogonal, i.e. TT'=I. 
This is implemented in line 140 of pointnet.py
original:   loss = torch.mean(torch.norm(torch.bmm(trans, trans.transpose(2, 1) - I), dim=(1, 2)))
new:         loss = torch.mean(torch.norm(torch.bmm(trans, trans.transpose(2, 1)) - I, dim=(1, 2)))
The original is essentially minimizing ||T(T'-I)|| while I suggest to replace with the new one which is minimizing ||TT'-I||
